platforms/core-runtime/internal-instrumentation-processor/src/main/java/org/gradle/internal/instrumentation/extensions/property/PropertyUpgradeAnnotatedMethodReader.java

```diff
@@ -649,6 +649,13 @@ private List<RequestExtra> getJvmRequestExtras(AccessorSpec accessor, Executable
             binaryCompatibility,
             accessor.bridgedMethod
         ));
+        if (accessor.accessorType == AccessorType.GETTER) {
+            // TODO Use a better class name here
+            extras.add(new PropertyUpgradeGetterOverrideRequestExtra(
```

Comment:
❓ We should probably have some flag to generate differentiate between `@ToBeReplacedByLazyProperty` and `@ReplacesEagerProperty` annotations, right?